src/NuGet.Core/NuGet.Commands/RestoreCommand/DependencyGraphResolver.cs

```diff
@@ -1324,15 +1316,42 @@ .. chosenResolvedItem.Suppressions
                         continue;
                     }
 
+                    // See if a dependency with the same version and suppressions has already been chosen
```

Comment:
Wouldn't it be more correct to say that we're checking that there's no suppressions?

Reply:
Updated